### PR TITLE
RA-92: Changed output data from Double to Integer to maintain consistency

### PR DESCRIPTION
**Pull request by Gennadii-T**

### CODE REVIEW

File: api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java [hunk 1]
```diff
@@ -220,7 +220,7 @@ public Double calculateTxSize(NetworkIdentifierType networkIdentifierType,
 
     String transaction = buildTransaction(unsignedTransaction.bytes(), signaturesList,
         unsignedTransaction.metadata());
-    return ((double) transaction.length() / 2);
+    return (transaction.length() / 2);
```
Comment (shleger) [inline api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java:223]: Why we divide transaction size and what happens if tx size is odd?

Comment (Gennadii-T) [reply api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java:223]: @Kammerlo could you please clarify this? I'm not very familiar with the original logic

Comment (shleger) [reply api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java:223]: As in the return statement `(transaction.length() / 2);` we should divide transaction length on two, why?
And why we can replace double with int in this case?

Comment (Kammerlo) [reply api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java:223]: It's not quite obvious. But the String represents Hexdata. So each Charakter is one hex char which is 4Bit. To get the size in bytes you have to divide the hexstring by two. I'll add a comment in your Branch to make it clearer for the next one to stumble across this line

**Review by Kammerlo — APPROVED** [review]

**Pull request merged**